keadm/cmd/keadm/app/cmd/edge/batchjoin.go

```diff
@@ -0,0 +1,179 @@
+package edge
+
+import (
+	"context"
+	"fmt"
+	"io/ioutil"
+	"os"
+	"sync"
+
+	"github.com/spf13/cobra"
+	"golang.org/x/crypto/ssh"
+	"gopkg.in/yaml.v3"
+)
+
+type Node struct {
+	IP              string `yaml:"ip"`
+	EdgeNodeName    string `yaml:"EdgeNodeName"`
+	Architecture    string `yaml:"architecture"`
+	Username        string `yaml:"username"`
+	Password        string `yaml:"password"`
+	KubeedgeVersion string `yaml:"version"`
+}
+
+type KeadmParams struct {
+	CloudCoreIPPort string `yaml:"cloudcore-ipport"`
+	Token           string `yaml:"token"`
+	KubeedgeVersion string `yaml:"kubeedge-version"`
+}
+
+type Config struct {
+	KeadmParams KeadmParams `yaml:"keadm_params"`
+	Nodes       []Node      `yaml:"nodes"`
+}
+
+var configFile string
+
+// NewDeprecatedEdgeJoin returns KubeEdge edge join command.
```

Comment:
```suggestion
// NewEdgeBatchJoin returns KubeEdge batch edge join command.
```

Reply:
> 

Thank you very much for your suggestions on the details of the code. I'll make the changes and improve them.